core/src/test/java/com/datastax/oss/driver/internal/core/cql/CqlRequestHandlerSpeculativeExecutionTest.java

```diff
@@ -101,34 +101,28 @@ public void should_schedule_speculative_executions(
       node1Behavior.verifyWrite();
       node1Behavior.setWriteSuccess();
 
-      harness.nextScheduledTask(); // Discard the timeout task
+      harness.nextScheduledTimeout(); // Discard the timeout task
 
-      ScheduledTaskCapturingEventLoop.CapturedTask<?> firstExecutionTask =
-          harness.nextScheduledTask();
-      assertThat(firstExecutionTask.getInitialDelay(TimeUnit.MILLISECONDS))
-          .isEqualTo(firstExecutionDelay);
+      Timeout speculativeExecution1 = harness.nextScheduledTimeout();
       Mockito.verifyNoMoreInteractions(nodeMetricUpdater1);
-      firstExecutionTask.run();
+      speculativeExecution1.task().run(speculativeExecution1);
```

Comment:
This looks a bit weird but that comes from the Netty API, nothing we can do about it in this PR.

Reply:
For the tasks we schedule on the timer, the Timeout argument to the `run()` call in the tests doesn't matter (it could be `null`). But that is only because the TimerTasks we schedule in the Handlers simply don't reference the Timeout in the runnable block of code. If that runnable block of code ever does, then passing `null` here could potentially throw an NPE, so I opted to pass in a known non-null reference.
But yes, it was (and still looks) weird.